web/api/v1/api_test.go

```diff
@@ -431,6 +444,18 @@ func TestEndpoints(t *testing.T) {
 						Health:           "unknown",
 					},
 				},
+				DroppedTargets: []*Target{
+					{
+						DiscoveredLabels: map[string]string{
+							"__address__":      "http://dropped.example.com:9115",
+							"__metrics_path__": "/probe",
+							"__scheme__":       "http",
+							"job":              "blackbox",
+						},
+						Labels: map[string]string{},
+						Health: "unknown",
```

Comment:
It doesn't make sense for a dropped target to have a health field.

Reply:
Agreed. I'm not sure a new struct definition is appropriate though, maybe we can just adjust the struct to omitempty and not set this field at all for dropped targets. Feels slightly hacky though. What's your suggestion?